spec/mailers/previews/subscription_mailer_preview.rb

```diff
@@ -3,7 +3,7 @@ class SubscriptionMailerPreview < ActionMailer::Preview
   def edit
     SubscriptionMailer.with(
       subscription: SubscriptionSession.first,
-      email: "test@gmail.com"
+      email: 'test@gmail.com'
```

Comment:
If this is just for testing purposes, let's use `example.com`

Reply:
👍